electronjs/readme.md

```diff
@@ -9,8 +9,8 @@
 ### Installation
 
 ```bash
-$> npm install fillr-extension --registry https://api.bintray.com/npm/fillr/npm
-$> npm install
+$> npm i @fillr_letspop/desktop-autofill
```

Comment:
maybe explain a little how where to get the username and password? or that they should contact us?

Reply:
@hades-6  @fillr_letspop/desktop-autofill is public, will add permission note for @fillr_letspop/cart-scraper which is private repo